htdocs/cron/class/cronjob.class.php

```diff
@@ -1223,6 +1223,45 @@ public function run_jobs($userlogin)
 
 		// Run a method
 		if ($this->jobtype == 'method') {
+			// Deny to launch a method from a deactivated module
```

Comment:
On each job, there is a property "module_name" loaded with the fetch()
Knowing if the module is on can be done more easily with
`if (!empty($this->module_name) && isModEnabled($this->module_name)) {`